participant/app/routes/index.tsx

```diff
@@ -4,15 +4,35 @@ import { Trans, useTranslation } from "react-i18next";
 
 export default function Index() {
   const { t } = useTranslation();
+  const listUserKeys: string[] = ["ifLink", "ifReceivesWIC", "ifThirtyDays"];
 
   return (
     <div>
       <h1>{t("Index.title")}</h1>
       <div className="font-sans-lg">
-        <Trans i18nKey="Index.intro" />
+        <p>
+          <Trans i18nKey="Index.intro" />
+        </p>
       </div>
       <div>
-        <Trans i18nKey="Index.body" />
+        <p>
+          <Trans i18nKey="Index.eligible" />
+        </p>
+      </div>
+      <ol className="usa-list">
+        {listUserKeys.map((key: string) => (
+          <li key={key}>
+            <Trans i18nKey={`Index.${key}`} />
+          </li>
+        ))}
+      </ol>
```

Comment:
Not entirely over how there is no `<List>` component

Can we make a followup ticket to create a <List> component and replace uses of `<ol>` and `<ul>` ?

Reply:
Created https://wicmtdp.atlassian.net/browse/PRP-252